backend/database/seeding/helpers.go

```diff
@@ -245,6 +254,38 @@ func newUserOperationPreferences(user models.User, op models.Operation, isFavori
 	}
 }
 
+func newUserGroupGen(first int64) func(name string, deleted bool) models.UserGroup {
+	id := iotaLike(first)
+	return func(name string, deleted bool) models.UserGroup {
+		if deleted {
+			now := internalClock.Now()
+			return models.UserGroup{
+				ID:        id(),
+				Slug:      name,
+				Name:      name,
+				CreatedAt: internalClock.Now(),
+				DeletedAt: &now,
+			}
+		} else {
+			return models.UserGroup{
+				ID:        id(),
+				Slug:      name,
+				Name:      name,
+				CreatedAt: internalClock.Now(),
+			}
+		}
+
+	}
+}
+
+func newUserGroupMapping(userID int64, groupID int64) models.UserGroupMap {
```

Comment:
This is totally fine, but in the future you might want to use models.User and models.UserGroup. The benefit is very minor -- using UserHarry instead of UserHarry.id.

Reply:
oh yeah now I see that most of the other helpers that use IDs take the full object

Reply:
changed